package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "better-commits",
-  "version": "1.5.1",
+  "version": "1.5.2",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "better-commits",
-      "version": "1.5.1",
+      "version": "1.5.2",
```

Comment:
Not something you have to change, just for reference. better-commits uses semantic versioning actions so this happens automatically when you merge to main.